CMakeLists.txt

```diff
@@ -1226,10 +1226,11 @@ if(CONFIG_GEN_ISR_TABLES)
   # isr_tables.c is generated from ${ZEPHYR_LINK_STAGE_EXECUTABLE} by
   # gen_isr_tables.py
   add_custom_command(
-    OUTPUT isr_tables.c
+    OUTPUT isr_tables.c isr_tables_vt.ld isr_tables_swi.ld
     COMMAND ${PYTHON_EXECUTABLE}
     ${ZEPHYR_BASE}/scripts/build/gen_isr_tables.py
     --output-source isr_tables.c
+    --linker-output-files isr_tables_vt.ld isr_tables_swi.ld
```

Comment:
>  Do you have any examples of samples which actually produces something in those files ?

answering my own question, in case others are wondering the same.

It appears that in order to make use of this new feature, one must set: `CONFIG_ISR_TABLES_LOCAL_DECLARATION=y`.